src/services/builder.js

```diff
@@ -113,6 +113,34 @@ angular.module('schemaForm').provider('sfBuilder', ['sfPathProvider', function(s
         }
       }
     },
+    //This is gross, but kind of clean at the same time...
+    oneOfTransclusion: function(args) {
```

Comment:
Method that allows one-of values to be transcluded into the DOM.
Sets the value of the ng-show value to be true if the parent object is selected.